ddtrace/opentelemetry/span.go

```diff
@@ -164,31 +164,31 @@ func (s *span) SetStatus(code otelcodes.Code, description string) {
 
 // SetAttributes sets the key-value pairs as tags on the span.
 // Every value is propagated as an interface.
+// Some attribute keys are reserved and will be remapped to Datadog reserved tags.
+// The mapping for reserved tags is as follows:
```

Comment:
`service.name` `resource.name`, and `span.type` are still reserved by Datadog, right? (If not, they should be) We should still document them here, even if it's not special to OTel.

Reply:
Gotcha